StreamChat.xcodeproj/project.pbxproj

```diff
@@ -4031,6 +4034,38 @@
 			path = CheckboxControl;
 			sourceTree = "<group>";
 		};
+		AD4474C1263AFD380030E583 /* Attachments */ = {
+			isa = PBXGroup;
+			children = (
+				ADB4164F26208C6B00E623E3 /* AttachmentPreview */,
+				AD4474C3263AFD680030E583 /* DocumentAttachments */,
+				AD4474C2263AFD5E0030E583 /* ImageAttachments */,
+			);
+			path = Attachments;
+			sourceTree = "<group>";
+		};
+		AD4474C2263AFD5E0030E583 /* ImageAttachments */ = {
+			isa = PBXGroup;
+			children = (
+				22C2350D259A25A300DC805A /* ChatImageAttachmentsCollectionView.swift */,
+				ADEA7F06261D23B500CA2289 /* ChatImageAttachmentsCollectionView_Tests.swift */,
+				228DC80D25704B410080A49F /* ChatImageAttachmentsCollectionViewCell.swift */,
+				AD4474FC263B19F90030E583 /* ChatImageAttachmentView.swift */,
+			);
+			path = ImageAttachments;
+			sourceTree = "<group>";
+		};
+		AD4474C3263AFD680030E583 /* DocumentAttachments */ = {
+			isa = PBXGroup;
+			children = (
+				22C23505259A257B00DC805A /* ChatDocumentAttachmentsCollectionView.swift */,
+				22086B37259509350007F8C0 /* ChatDocumentAttachmentsCollectionViewCell.swift */,
+				22086B47259509AC0007F8C0 /* ChatDocumentAttachmentsCollectionViewLayout.swift */,
+				22086B3F259509450007F8C0 /* ChatDocumentAttachmentView.swift */,
+			);
+			path = DocumentAttachments;
+			sourceTree = "<group>";
+		};
```

Comment:
This makes me wonder if we move the attachments views of the composer to the common folder, where will the attachments view of the message list live? 🤔 CC @VojtaStavik @evsaev 

We kinda need to come up with something to distinguish them. If we have `ChatImageAttachmentView` for the composer image attachments item view, what will be the name of the attachment view in the message list?